### ISSUE TITLE
2.6.2 will fail to start if you are not using nvm

### ISSUE THREAD

**Issue by dustinrue** [10up/wp-local-docker-v2#78]
**Describe the bug**
When using node version manager, node is installed into the user home directory. The user home directory is "shared" by default in Docker Desktop meaning any file in the home directory can be mapped into a running container. 

However, if the user is not using nvm and is instead installing nodejs using a tool like brew then it will be installed into /usr/local which is not shared. Starting the MySQL container will fail in this case unless the user adjusts their Docker Desktop configuration

**Steps to Reproduce**

See above

**Expected behavior**
No failures

**Screenshots**

**Environment information**
 - Device: a Mac for sure
 - OS: all are affected

**Additional context**

Two potential "solutions." Advise the the user to use nvm or to adjust their Docker Desktop configuration.

**Comment by tlovett1** [comment]
I can't get WP Local Docker to start at all @christianc1 . I am using NVM:

```
taylor$ 10updocker start
Locating project files for test
Ensuring global network exists
 - Network exists
Ensuring global cache volume exists
 - Creating Volume
Ensuring global services are running
Starting global_mysql_1 ...
global_dns_1 is up-to-date
global_gateway_1 is up-to-date
Starting global_mysql_1 ... error

ERROR: for global_mysql_1  Cannot start service mysql: b'Mounts denied: \r\nThe path /usr/local/lib/node_modules/wp-local-docker/global/config/mysql/config.cnf\r\nis not shared from OS X and is not known to Docker.\r\nYou can configure shared paths from Docker -> Preferences... -> File Sharing.\r\nSee https://docs.docker.com/docker-for-mac/osxfs/#namespaces for more info.\r\n.'

ERROR: for mysql  Cannot start service mysql: b'Mounts denied: \r\nThe path /usr/local/lib/node_modules/wp-local-docker/global/config/mysql/config.cnf\r\nis not shared from OS X and is not known to Docker.\r\nYou can configure shared paths from Docker -> Preferences... -> File Sharing.\r\nSee https://docs.docker.com/docker-for-mac/osxfs/#namespaces for more info.\r\n.'
ERROR: Encountered errors while bringing up the project.
(node:1699) UnhandledPromiseRejectionWarning: Error: Command failed: docker-compose up -d
    at checkExecSyncError (child_process.js:603:11)
    at execSync (child_process.js:640:13)
    at startGateway (/usr/local/lib/node_modules/wp-local-docker/src/gateway.js:114:5)
    at Object.startGlobal (/usr/local/lib/node_modules/wp-local-docker/src/gateway.js:138:11)
(node:1699) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:1699) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by christianc1** [comment]
Docker is attempting to mount the MySQL configuration in the npm package on the container and due to Docker Desktop settings and the location to which you've installed node is the issue.

In Docker Desktop -> Preferences -> File Sharing, you should be able to add:
`/usr/local/lib/node_modules`

**Comment by dustinrue** [comment]
That’s what I am thinking. Move the global info and config into the user home directory. I had gotten started on the move recently and should be able to concentrate on it soon.

**Comment by dustinrue** [comment]
This is resolved in #83
